### PR TITLE
reset devnet and set 30M epoch for all network except mainnet/testnet

### PR DESCRIPTION
**Pull request by sophoah**
This PR is preparing for the devnet reset by :
- updating the different epoch feature activation
- updating the devnet instance configuration

and also update the epoch activation for 30M block gas for all network except mainnet and testnet

### CODE REVIEW

File: internal/configs/sharding/partner.go [hunk 1]
```diff
@@ -44,10 +44,6 @@ func (ps partnerSchedule) InstanceForEpoch(epoch *big.Int) Instance {
 	switch {
 	case params.PartnerChainConfig.IsHIP30(epoch):
 		return partnerV4
-	case params.PartnerChainConfig.IsFeeCollectEpoch(epoch):
-		return partnerV3
-	case epoch.Cmp(feeCollectEpochV1) >= 0:
-		return partnerV2
```
Comment (GheisMohammadi) [inline internal/configs/sharding/partner.go:50]: do we need to remove the checkings for V2 and V3? Jumping from V1 to V4?
I think keeping these lines helps to see other versions and users could see the path from V1 to V4.

**Comment by GheisMohammadi** [comment]
The Travis tests are failed in this PR because the **checkGetReceiptsResult** redeclared in same package. I fixed this in PR #4518

File: internal/configs/sharding/partner.go (hunk 1 above)
Comment (sophoah) [reply internal/configs/sharding/partner.go:50]: this section doesn't check anything it just return the instance configuration ie `InstanceForEpoch` to be used
v2 basically implemented hip28 partially when we only had one multisig to send the fee collection.
v3 implemented the full hip29 that contained both multisig (infra and community)
i kept partnerV2 and partnerV3 for historical reason so we can go back on it eventually.

Comment (GheisMohammadi) [reply internal/configs/sharding/partner.go:50]: developers look at this part and can't see any historical reason for jumping from V1 to V4. This is the reason we should keep that part. I don't see any benefits of removing it.

**Review by Frozen — APPROVED** [review]

File: internal/configs/sharding/partner.go (hunk 1 above)
Comment (sophoah) [reply internal/configs/sharding/partner.go:50]: but there is also no business benefits of keeping instance configuration. Ok you know what, I'll simply delete and replace V4 with V2 :)

**Review by GheisMohammadi — APPROVED** [review]

**Review by sunwavesun — APPROVED** [review]

**Comment by sophoah** [comment]
rebased the PR with the latest dev

**Pull request merged**